Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift

```diff
@@ -236,6 +236,11 @@ public enum RenderBlockContent: Equatable {
     public struct Table: Equatable {
         /// The style of header in this table.
         public var header: HeaderType
+        /// The text alignment of each column in this table.
+        ///
+        /// A `nil` value for this property is the same as all the columns being
+        /// ``RenderBlockContent/ColumnAlignment/unset``.
```

Comment:
This could be inferred as this type's Equatable conformance implementing this behavior. Maybe we can mention that this is the behavior _in practice_ or something like that.

Reply:
Or we can actually override `==` and implement the behavior there.

Reply:
Should a Table (with 3 rows and alignments are [.unset, .unset, .unset]) equal to a Table (with the same 3 rows but alignments are nil)?

Their actual behavior is the same but the current default Equatable implementation is false.

Reply:
I've pushed up a new commit that implements this behavior into the Equatable conformance, and adds some tests to ensure that the behavior holds.